### ISSUE TITLE
Homebrew Role: Permissions issue on /usr/local/man/man5

### ISSUE THREAD

**Issue by ahgraber** [geerlingguy/ansible-collection-mac#24]
When I try to install a list of packages with the homebrew role, I end up with a series of errors about file ownership of `/usr/local/man/man5`.  

Screengrab attached

<img width="703" alt="Screen Shot 2021-11-02 at 10 34 25 AM" src="https://user-images.githubusercontent.com/24922003/139868080-46368db5-c1d2-41b0-9f43-40f60f91f8fa.png">
ed

my homebrew_vars.yaml

```yaml
---
homebrew_install_path: /usr/local/homebrew
homebrew_installed_packages:
  - autoconf
  - automake
  - bat
  - coreutils
  - curl
  - docker
  - exa
  - gzip
  - htop
  - jq
  - libtool
  - libxslt
  - libyaml
  - npm
  - pkg-config
  - tmux
  - unar
  - unixodbc
  - unzip
  - vim
  - wget
  - xz
  - yarn
  - zlib
  - zsh

homebrew_cask_appdir: /Applications
homebrew_cask_apps:
  - appcleaner
  - iterm2
  - osxfuse
  - oversight
  - postman
  - spotify
  - visual-studio-code
  - vlc

homebrew_taps:
  - homebrew/cask
  - homebrew/cask-versions
  - name: microsoft/git
    url: 'https://github.com/microsoft/homebrew-git'

```

my playbook

```yaml
---
- hosts: localhost

  vars_files:
    - vars/homebrew_vars.yaml

  tasks:
    - name: homebrew
      include_role:
        name: geerlingguy.mac.homebrew
```

My ansible-playbook command

```sh
ansible-playbook playbook.yaml -i inventory -v
```

**Comment by ahgraber** [comment]
It looks like merging #21 in would resolve this as well.  If I manually run `chown -R $(whoami) /usr/local/share`, everything runs fine.

**Comment by geerlingguy** [comment]
Merged #21 and am tagging a new release now. Please let me know if that fixes the issue.

**Comment by ahgraber** [comment]
This appears to work!  Thanks (both for the quick response and for the work you do for the community)!
